packages/battery/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Your Company. All rights reserved.
```

Comment:
Also generally missing copyright header in the .dart|.h|.m|.java files that "links" to this LICENSE file